app/controllers/admin/api/services/proxy/configs_controller.rb

```diff
@@ -30,11 +30,11 @@ class Admin::Api::Services::Proxy::ConfigsController < Admin::Api::Services::Bas
   ##~ op.parameters.add @parameter_environment
   #
   def index
-    respond_with(service_proxy_configs)
+    respond_with service_proxy_configs.order(:id).paginate(pagination_params)
```

Comment:
is `order(:id)` without specifying the direction the same for all DBs?

Reply:
It should be by default `asc` everywhere 🤔 But I will add it just in case and for clarify 👍🏼